src/core/etl/src/Flow/ETL/ExternalSort/CacheExternalSort.php

```diff
@@ -34,7 +34,7 @@ public function sortBy(EntryReference ...$refs) : Extractor
     {
         /** @var array<string, \Generator<Rows>> $cachedPartsArray */
         $cachedPartsArray = [];
-        $maxRowsSize = 0;
+        $maxRowsSize = 1;
```

Comment:
could you elaborate on this one? I don't remember anymore that logic and would need to go through the code again to understand why it was set to 0 and why it's even needed

Reply:
This code was IMO false positive, cause as the chunk size was always: `int<1, max>`, and never `0`, this value was always overwritten by a later loop which has: `$maxRowsSize = \max($maxRowsSize, $rows->count());`.

Now with a clear declaration of what chunk size is, it was reported by static analysis that this value is wrong & should never happen.